src/node_dir.h

```diff
@@ -46,7 +45,7 @@ class DirHandle : public AsyncWrap {
   void GCClose();
 
   uv_dir_t* dir_;
-  uv_dirent_t dirent_;
+  uv_dirent_t dirents_[32];
```

Comment:
nit: maybe a little comment on this choice.

Reply:
@devnexen I’ve added a comment describing the impact of this number, but the choice is really just one that I’ve made according to my intuition and not based on anything empiric.

Reply:
Happy with it.